_developer-files/connect/api/objects/form-properties/sources/saas/saasoptics-object.md

```diff
@@ -0,0 +1,70 @@
+---
+# -------------------------- #
+#     USING THIS TEMPLATE    #
+# -------------------------- #
+
+## NEED HELP USING THIS TEMPLATE? SEE:
+## https://docs-about-stitch-docs.netlify.com/reference/connect-templates/destination-form-property/
+## FOR INSTRUCTIONS & REFERENCE INFO
+
+
+# -------------------------- #
+#        CONTENT TYPE        #
+# -------------------------- #
+
+product-type: "connect"
+content-type: "api-form"
+form-type: "source"
+key: "source-form-properties-saasoptics-object"
+
+
+# -------------------------- #
+#        OBJECT INFO         #
+# -------------------------- #
+
+title: "SaaSOptics Source Form Property"
+api-type: "platform.saasoptics"
+display-name: "SaaSOptics"
+
+source-type: "saas"
+docs-name: "saasoptics" # This should be whatever integration.name is. Ex: LinkedIn Ads is linkedin-ads
+
+
+# -------------------------- #
+#      OBJECT ATTRIBUTES     #
+# -------------------------- #
+
+uses-start-date: true
+
+# Only source-specific attributes need to be listed here.
+# The following attributes are considered common,
+# and therefore don't need to be listed:
+# anchor_time, cron_expression, frequency_in_minutes, image_version, start_date 
+
+object-attributes:
+  - name: "account_name"
+    type: "string"
+    required: true
+    description: |
+      The name of your unique SaaSOptics database schema, called an "account." This account is the one in which your subscription financial records are managed.
+    value: "<YOUR_ACCOUNT_NAME>"
+
+  - name: "server_subdomain"
+    type: "string"
+    required: true
+    description: "The subdomain of the SaaSOptics server where the account is running."
```

Comment:
Same here as the `account_name` comment I left. No need to overcomplicate this - it's just as simple as `"The subdomain of your {{ form-property.display-name }} account."` (I see you took this from SaaSOptics' docs, which is good, but for our purposes I think their descriptions are a bit overboard.)

I *would* add, however, that the user shouldn't include the `.saasoptics.com` portion of their subdomain. If they do, the connection will fail because the tap expects just the first part of the string (ex: `stitch`) and not the entire URL (`stitch.saasoptics.com`).

Also, remember to use `{{ form-property.display-name }}` consistently.